### PR TITLE
OHP updates to search + filtering and others, styling pending

### PR DESCRIPTION
**Pull request by erbnando**

### CODE REVIEW

**Comment by cfhull** [comment]
Couple minor issues
- Add some padding between and below the date pickers
- Clear the date inputs when clicking clear filters
- I think it might make more sense to have `Start Date` and `End Date` as labels for each date picker.
image

**Review by cfhull — CHANGES_REQUESTED** [review]
Code looks good. I left some comments on UI.

File: src/components/stories/filters/StorySearchContainer.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      props.searchResults(items);
+    }
+  };
+
+  return (
+    <TextField
+      autoFocus={true}
+      className={classes.search}
+      onChange={searchHandler}
+      placeholder="Search by keyoword or name"
```
Comment (cfhull) [inline src/components/stories/filters/StorySearchContainer.js:51]: small typo `keyoword`

File: src/components/stories/filters/StorySearchData.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import { useMemo } from "react";
+import * as JsSearch from "js-search";
+import { useQuery } from "react-query";
+
+const indexedFields = ["date", "keywords", "participantName", "participantDescription", "interviewText", "interviewTextCont"];
+
+export default function StorySearchData(posts) {
+
+  const { data } = useQuery('posts', () => {return posts})
```
Comment (cfhull) [inline src/components/stories/filters/StorySearchData.js:9]: I'm probably forgetting something, but what is the reason to use `react-query` here?

Comment (erbnando) [reply src/components/stories/filters/StorySearchData.js:9]: Wasn't sure what to use for search, so i mimicked site search, which uses react-query to format searchable data -- see here: 

https://github.com/Hyperobjekt/covid-19-behind-bars/blob/oral-history-project/src/components/search/useSearchData.js#L9

Which might be overkill/unnecessary in this case -- I doubt input and output are very different in this case.

Comment (erbnando) [reply src/components/stories/filters/StorySearchData.js:9]: of course the heavy lifting here is done by js-search

**Comment by erbnando** [comment]
@cfhull,

- ~~Add some padding between and below the date pickers~~ DONE
- ~~Clear the date inputs when clicking clear filters (ignore this if it's impossible)~~ DONE
- I think it might make more sense to have Start Date and End Date as labels for each date picker -- this is per mockups, see below:

image

**Comment by erbnando** [comment]
i'm aware there's tons of styling missing, and that's what i'm working on next

File: src/components/stories/filters/StorySearchContainer.js (hunk 1 above)
Comment (erbnando) [reply src/components/stories/filters/StorySearchContainer.js]: fixed

**Pull request merged**